pinconfigs/sam-r34-r35-b.yml

```diff
@@ -0,0 +1,340 @@
+# Copyright (c) 2022 Argentum Systems Ltd.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Sources:
+# - SAM-R34-R35-Low-Power-LoRa-Sub-GHz-SiP-Data-Sheet-DS70005356C.pdf (Revision C - 05/2019)
+#
+# Pin codes:
+#
+# - 64      pins: j
+#
+# SoC Revision (variant) supported:
+#
+# - SAMR34/SAMR35
+# - B = Release to Production
+
+model: atmel,sam
+
+family: r34-r35
+
+map: SAM_PINMUX
+
+series: [r34, r35]
+
+revisions:
+  r34: b
+  r35: b
+
```

Comment:
No need to add revisions for this variant.

Reply:
done